src/components/Captures/CaptureGallery.test.js

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import {
+  act,
+  render,
+  screen,
+  within,
+  cleanup,
+  waitForElementToBeRemoved,
+} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import captureApi from '../../api/treeTrackerApi';
+import theme from '../common/theme';
+import { ThemeProvider } from '@material-ui/core/styles';
+import { AppProvider } from '../../context/AppContext';
+import { CapturesContext } from '../../context/CapturesContext';
+import { GrowerContext } from '../../context/GrowerContext';
+import { SpeciesContext } from '../../context/SpeciesContext';
+import { TagsContext } from '../../context/TagsContext';
+import CaptureGallery from './CaptureGallery';
+import {
+  CAPTURE,
+  CAPTURES,
+  GROWER,
+  ORGS,
+  TAG,
+  TAGS,
+  SPECIES,
+  capturesValues,
+  growerValues,
+  tagsValues,
+  speciesValues,
+} from '../tests/fixtures';
+
+import * as loglevel from 'loglevel';
+const log = loglevel.getLogger('../tests/verify.test');
+
+jest.setTimeout(7000);
+jest.mock('../../api/treeTrackerApi');
+
+describe('Captures', () => {
+  // mock captures context api methods
+  const getCaptures = () => {
+    log.debug('mock getCaptures:');
+    return Promise.resolve(CAPTURES);
+  };
+  const getCaptureCount = () => {
+    log.debug('mock getCaptureCount:');
+    return Promise.resolve({ count: 4 });
+  };
+
+  // mock the treeTrackerApi
+  // let captureApi = require('../../api/treeTrackerApi').default;
+
+  // captureApi.getCaptureById = (_id) => {
+  //   log.debug('mock getCaptureById:');
+  //   return Promise.resolve(CAPTURE);
+  // };
+  // captureApi.getSpecies = () => {
+  //   log.debug('mock getSpecies:');
+  //   return Promise.resolve(SPECIES);
+  // };
+  // captureApi.getSpeciesById = (_id) => {
+  //   log.debug('mock getSpeciesById:');
+  //   return Promise.resolve(SPECIES[0]);
+  // };
+  // captureApi.getCaptureCountPerSpecies = () => {
+  //   log.debug('mock getCaptureCountPerSpecies:');
+  //   return Promise.resolve({ count: 7 });
+  // };
+  // captureApi.getTags = () => {
+  //   log.debug('mock getTags:');
+  //   return Promise.resolve(TAGS);
+  // };
+  // captureApi.getTagById = (_id) => {
+  //   log.debug('mock getTagById:');
+  //   return Promise.resolve(TAG);
+  // };
+  // captureApi.getOrganizations = () => {
+  //   log.debug('mock getOrganizations:');
+  //   return Promise.resolve(ORGS);
+  // };
```

Comment:
If the commented code is not needed, maybe we can remove it. If it's for a future use, I think a comment could help.

Reply:
It was part of experimental mocking, I'll remove